libgadget/gravpm.c

```diff
@@ -203,10 +203,6 @@ static int pm_mark_region_for_node(int startno, int rid, int * RegionInd, const
                 int p = nop->s.suns[i];
                 RegionInd[p] = rid;
 #ifdef DEBUG
-                /* when we are in PM, all particles must have been synced. */
```

Comment:
I suggest preserving a weaker version of this, e.g. all Ti_drift is identical to P[0]'s.

Reply:
I moved the check to rebuild_activelist, so that it is run on all timesteps and is a strictly stronger check. I also dropped the DEBUG.